CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Change default value of `-blocks-storage.bucket-store.index-cache.memcached.max-async-concurrency` from `50` to `3` #6265
```

Comment:
Oops. I think the changelog is wrong. You changed `-blocks-storage.bucket-store.index-cache.multilevel.max-async-concurrency` not `-blocks-storage.bucket-store.index-cache.memcached.max-async-concurrency`.

Let's create a new PR to change all async concurrency to 3, including Redis cache

Reply:
Thank you for catching my mistake. I will be making another PR.